src/app/shared/button/button.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-submit',
```

Comment:
Faltou alterar o selector, uma sugestão:

```suggestion
  selector: 'shared-button',
```

Reply:
Feito